docs/src/modules/components/AppContent.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React from 'react';
+import * as React from 'react';
```

Comment:
@rosskevin Won't that break with users using Typescript or no using the babel flow plugin?